shell/platform/darwin/macos/framework/Source/FlutterEngineTest.mm

```diff
@@ -805,55 +805,75 @@ - (nonnull NSView*)createWithViewIdentifier:(int64_t)viewId arguments:(nullable
   EXPECT_TRUE(announced);
 }
 
-TEST_F(FlutterEngineTest, RunWithEntrypointUpdatesDisplayConfig) {
```

Comment:
Presumably, it wasn't the intention of this PR to remove this test and the next one? Probably just a bad merge?

Reply:
Huh.  No, definitely not intended.